### PR TITLE
JNoSQL loading Entities

### PR DESCRIPTION
**Pull request by mswatosh**
Adding JNoSQL project and loading Jakarta Data Entities into a Servlet. This is a start, but not functional (can't save entities).

### CODE REVIEW

File: dev/io.openliberty.data.internal.nosql/.classpath [hunk 1]
```diff
@@ -3,5 +3,70 @@
 	<classpathentry kind="src" path="src"/>
 	<classpathentry kind="con" path="aQute.bnd.classpath.container"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
+	<classpathentry kind="lib" path="/io.openliberty.jakarta/generated/io.openliberty.jakarta.cdi.4.0.jar">
```
Comment (njr-11) [inline dev/io.openliberty.data.internal.nosql/.classpath]: Was this part of the classpath intended to be checked in?

File: dev/io.openliberty.data.internal.nosql/src/io/openliberty/data/internal/nosql/cdi/DocumentExtensionProvider.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+           service = { CDIExtensionMetadata.class, DocumentExtensionProvider.class, ApplicationMetaDataListener.class })
+public class DocumentExtensionProvider implements CDIExtensionMetadata, ApplicationMetaDataListener {
+    private static final TraceComponent tc = Tr.register(DocumentExtensionProvider.class);
+
+    private static final Set<Class<? extends Extension>> extensions = Collections.singleton(DocumentExtension.class);
+
+    @Override
+    @Trivial
+    public void applicationMetaDataCreated(MetaDataEvent<ApplicationMetaData> event) throws MetaDataException {
+        // TODO Auto-generated method stub
```
Comment (njr-11) [inline dev/io.openliberty.data.internal.nosql/src/io/openliberty/data/internal/nosql/cdi/DocumentExtensionProvider.java]: Does the ApplicationMetaDataListener portion of this do anything? Should it be removed or is it here for future use?

File: dev/io.openliberty.data.internal_fat_nosql/.classpath [hunk 3]
```diff
@@ -8,5 +8,20 @@
 			<attribute name="module" value="true"/>
 		</attributes>
 	</classpathentry>
+	<classpathentry kind="lib" path="/io.openliberty.jakarta.data.1.0/generated/io.openliberty.jakarta.data.1.0.jar" sourcepath="/io.openliberty.jakarta.data.1.0/src">
```
Comment (njr-11) [inline dev/io.openliberty.data.internal_fat_nosql/.classpath]: Another classpath entry to check whether should be included

**Review by njr-11 — DISMISSED** [review]
I added a couple of minor comments.  At some future point, we will need to switch to using jnosql as a library and having the server only provide the Jakarta NoSQL spec API.

**Review by njr-11 — DISMISSED** [review]

**Comment by mswatosh** [comment]
#build

**Comment by mswatosh** [comment]
#build

**Comment by mswatosh** [comment]
#build

**Review by njr-11 — APPROVED** [review]

**Comment by mswatosh** [comment]
#libby

**Pull request merged**